contributing_to_docs/docs_production_deployment.adoc

```diff
@@ -0,0 +1,51 @@
+[id="contributing-to-docs-docs-production-deployment"]
+= Production deployment of the OpenShift documentation
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+:linkattrs:
+:description: How to deploy the entire set of documentation
+
+toc::[]
+
+== Source-to-image pipeline
+OpenShift documentation is built and deployed on an https://cloud.redhat.com/products/dedicated/[OpenShift Dedicated cluster]
+using a https://github.com/openshift/source-to-image[source-to-image] build pipeline.
+
+The source-to-image builder image is built from a https://github.com/openshift-cs/docs-builder/[community project in GitHub]
+and published to https://quay.io/repository/openshift-cs/docs-builder.
+
+== Documentation deployment
+Deploying the OpenShift documentation is simplified by using a
+https://github.com/openshift-cs/docs-builder/blob/main/template.yaml[pre-built OpenShift template YAML].
+
+You can use the following command to deploy the OpenShift Container Platform (commercial) documentation
```

Comment:
s/documentation/documentation:

Reply:
Ha! I thought this was a spelling error...took way to much time to realize you wanted a `:` at the end 😉